curvefs/src/client/curve_fuse_op.cpp

```diff
@@ -263,20 +266,17 @@ void FuseOpGetXattr(fuse_req_t req, fuse_ino_t ino, const char *name,
     size_t size) {
     InflightGuard guard(&g_clientOpMetric->opGetXattr.inflightOpNum);
     LatencyUpdater updater(&g_clientOpMetric->opGetXattr.latency);
-    std::unique_ptr<char[]> buf(new char[MAXXATTRLENGTH]);
-    std::memset(buf.get(), 0, MAXXATTRLENGTH);
-    CURVEFS_ERROR ret = g_ClientInstance->FuseOpGetXattr(req, ino, name,
-                                                         buf.get(), size);
-    if (ret != CURVEFS_ERROR::OK && ret != CURVEFS_ERROR::NODATA) {
-        g_clientOpMetric->opGetXattr.ecount << 1;
-        FuseReplyErrByErrCode(req, ret);
-        return;
-    }
-
+    char buf[MAXXATTRLENGTH] = {0};
```

Comment:
use std::memset to clear buf,  = {0} is not work.

Reply:
= {0} works and test a demo.